AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -243,72 +240,96 @@ export class SummaryComponent implements OnInit, OnDestroy {
+                if (this.hasJudge(hearingDetailsResponse)) {
+                    // Poll Video-Api for booking confirmation
+                    const schedule = timer(0, 5000).subscribe(async counter => {
+                        const hearingStatusResponse = await this.hearingService.getStatus(hearingDetailsResponse.id);
+                        if (hearingStatusResponse?.success || counter === 10) {
+                            schedule.unsubscribe();
+                            await this.processBooking(hearingDetailsResponse, hearingStatusResponse);
+                        }
                     });
-
-                    const isMultipleIndividualHearingDates = this.hearing.hearing_dates && this.hearing.hearing_dates.length > 1;
-                    const isHearingDateRange = !this.hearing.hearing_dates || this.hearing.hearing_dates.length === 0;
-
-                    if (isMultipleIndividualHearingDates) {
-                        this.logger.info(`${this.loggerPrefix} Hearing has multiple, individual days. Booking remaining days`, {
-                            hearingId: hearingDetailsResponse.id,
-                            caseName: this.hearing.cases[0].name,
-                            caseNumber: this.hearing.cases[0].number
-                        });
-                        await this.hearingService.cloneMultiHearings(
-                            hearingDetailsResponse.id,
-                            new MultiHearingRequest({
-                                hearing_dates: this.hearing.hearing_dates.map(date => new Date(date))
-                            })
-                        );
-                    } else if (isHearingDateRange) {
-                        this.logger.info(`${this.loggerPrefix} Hearing has a range of days. Booking remaining days`, {
-                            hearingId: hearingDetailsResponse.id,
-                            caseName: this.hearing.cases[0].name,
-                            caseNumber: this.hearing.cases[0].number
-                        });
-                        await this.hearingService.cloneMultiHearings(
-                            hearingDetailsResponse.id,
-                            new MultiHearingRequest({
-                                start_date: new Date(this.hearing.scheduled_date_time),
-                                end_date: new Date(this.hearing.end_hearing_date_time)
-                            })
-                        );
-                    } else {
-                        this.logger.info(`${this.loggerPrefix} Hearing has just one day, no remaining days to book`, {
-                            hearingId: hearingDetailsResponse.id,
-                            caseName: this.hearing.cases[0].name,
-                            caseNumber: this.hearing.cases[0].number
-                        });
-                    }
+                } else {
+                    await this.processMultiHearing(hearingDetailsResponse);
```

Comment:
could we add test coverage for when there's no judge?

Reply:
test added